var/spack/repos/builtin/packages/gperftools/package.py

```diff
@@ -22,4 +22,18 @@ class Gperftools(AutotoolsPackage):
     version('2.4', sha256='982a37226eb42f40714e26b8076815d5ea677a422fb52ff8bfca3704d9c30a2d')
     version('2.3', sha256='093452ad45d639093c144b4ec732a3417e8ee1f3744f2b0f8d45c996223385ce')
 
-    depends_on("unwind")
+    variant('sized-delete', default=False, description="Build sized delete operator")
+    variant('dynamic-sized-delete-support', default=False, description="Try to build run-time switch for sized delete operator")
```

Comment:
Dashes in variant names are discouraged, see https://github.com/spack/spack/pull/24631#discussion_r662798100 @alalazo